.github/workflows/lint.yaml

```diff
@@ -26,5 +19,6 @@ jobs:
     runs-on: ubuntu-latest
     name: PR - Markdownlint
     steps:
+      - uses: actions/checkout@v4
```

Comment:
Without this we were doing nothing... :(

Reply:
I'm not sure I understand

Reply:
Nothing is checked out so nothing is actually linted.